packages/core/src/sheets/styles.ts

```diff
@@ -42,6 +51,10 @@ export class Styles {
     add(data: IStyleData): string {
         const id = Tools.generateRandomId(6);
         this._styles[id] = data;
+        // update cache
+        const styleObject = JSON.stringify(data);
```

Comment:
Should use LRU cache to avoid memory overflow.

Reply:
Will be fixed in other branches